src/app/sharedUI/screens/Artwork/Artwork.tsx

```diff
@@ -20,12 +40,104 @@ export const Artwork = () => {
     ),
   }))
 
+  const bottomSheetModalRef = useRef<BottomSheetModal>(null)
+
+  const snapPoints = useMemo(() => ["25%", "65%"], [])
+
+  const handlePresentModalPress = useCallback(() => {
+    bottomSheetModalRef.current?.present()
+  }, [])
+
+  const handleSheetChanges = useCallback((index: number) => {}, [])
+
+  const isEditArtworkHidden = GlobalStore.useAppState(
+    (state) => state.presentationMode.hiddenItems.editArtwork
+  )
+
   return (
-    <>
-      <Header safeAreaInsets />
+    <BottomSheetModalProvider>
+      <Header
+        safeAreaInsets
+        rightElements={
+          <Touchable onPress={handlePresentModalPress}>
+            <MoreIcon />
+          </Touchable>
+        }
+      />
       <Flex flex={1}>
         <ScrollableScreensView screens={screens} initialScreenName={params.slug} />
       </Flex>
-    </>
+      <BottomSheetModal
+        backdropComponent={() => (
+          <Touchable
+            style={{
+              opacity: 0.4,
+              width: "100%",
+              height: "100%",
+              position: "absolute",
+            }}
+            underlayColor="transparent"
+            onPress={() => bottomSheetModalRef.current?.close()}
+          >
+            <Flex background="black" height="100%" />
+          </Touchable>
+        )}
+        enablePanDownToClose
+        ref={bottomSheetModalRef}
+        index={1}
+        snapPoints={snapPoints}
+        onChange={handleSheetChanges}
+        handleComponent={() => (
+          <Flex position="relative" alignItems="center" justifyContent="center" height="50">
```

Comment:
By default flex has position: relative
```suggestion
          <Flex alignItems="center" justifyContent="center" height="50">
```